gtsfm/two_view_estimator.py

```diff
@@ -23,6 +26,8 @@
 pil_logger = logging.getLogger("PIL")
 pil_logger.setLevel(logging.INFO)
 
+ESSENTIAL_MATRIX_EPIPOLAR_DISTANCE_THRESHOLD = 0.1  # TODO: move to config?
```

Comment:
i prefer in config

Reply:
This is technically an evaluation related param though.

Reply:
evaluation is something to configure, though, since there's no one correct way to evaluate

Reply:
Makes sense. Although I'd like to somehow have two different configs (or sections) for building the pipeline and evaluating the pipeline.

Reply:
agreed, let's leave that as a to-do